extensions/interactions/NumericInput/NumericInput.html

```diff
@@ -3,18 +3,16 @@
 
 <script type="text/ng-template" id="interaction/NumericInput">
   <div class="row numeric-input-container">
-    <form ng-submit="submitAnswer(answer)" role="form" class="form-horizontal">
-      <div class="col-lg-12 col-md-12 col-sm-12">
-        <ng-form name="numericInputForm">
-          <schema-based-editor schema="NUMERIC_INPUT_FORM_SCHEMA" local-value="answer" label-for-focus-target="labelForFocusTarget">
-          </schema-based-editor>
-        </ng-form>
+    <div class="col-lg-12 col-md-12 col-sm-12">
+      <ng-form name="numericInputForm">
```

Comment:
Do you still need the ng-form? (Btw, was the 'form' stuff needed for checking validity, or do the validity checks still work as before once you've removed the 'form'? If the latter, everything's fine; if the former, might need to find a way to get them back -- maybe by using a form only for part of this instead.)